torchbox/torchbox.com
# Front-end for work page and historical work page

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1347772963)

### Description of Changes Made

Adds the front-end code for the work page and the historical work page. Adds a common base template for the shared elements

### How to Test

In the pattern library:

http://localhost:8000/pattern-library/pattern/patterns/pages/work/work_page.html
http://localhost:8000/pattern-library/pattern/patterns/pages/work/historical_work_page.html

In the build:
View an old work page:
http://localhost:8000/work/voyage-of-discovery-nasa-jpl/
Create a new work page to see a new one.

### Screenshots

<details>
  <summary>Expand to see more</summary>
![Uploading screencapture-localhost-8000-work-a-new-work-page-2024-01-29-18_29_25.png…]()

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
